doc/build.zig.zon.md

```diff
@@ -20,12 +20,31 @@ will stick with this provided value.
 It is redundant to include "zig" in this name because it is already within the
 Zig package namespace.
 
+Must be a valid bare Zig identifier (don't `@` me), limited to 32 bytes.
+
+### `id`
+
+Together with name, this represents a globally unique package identifier. This
+field should be initialized with a 16-bit random number when the package is
```

Comment:
I think "should" is too vague when other fields have a definitive "required" or "optional".

Reply:
Elsewhere, the word "must" and "required" indicate that an error will occur. Here, I think the word "should" is appropriate because there is no error for not doing the thing, instead there are social consequences.

Reply:
prior art https://datatracker.ietf.org/doc/html/rfc2119